design/admin/templates/content/parts/object_information.tpl

```diff
@@ -30,9 +30,9 @@
 <label>{'Modified'|i18n( 'design/admin/content/history' )}:</label>
 {if $object.modified}
 {$object.modified|l10n( shortdatetime )}<br />
-{foreach $object.versions as $version}
-{if eq($version.version, $object.published_version)}
-{$version.creator.name|wash}
+{foreach $object.versions as $tmp_version}
```

Comment:
nitpick, but could also be for instance `$object_version`

Reply:
There is a variable named $object_version used in a few places, so I would need to check if it doesn't cause a similar issue.

Reply:
Ok, lets stay with this.